Tools/TestWebKitAPI/cocoa/TestWebExtensionsDelegate.h

```diff
@@ -70,6 +70,9 @@
 
 @property (nonatomic, copy) void (^didUpdateSidebar)(_WKWebExtensionSidebar *);
 
+@property (nonatomic, copy) void (^createBookmarkWithParentIdentifier)(NSString * parentId, NSNumber * index, NSString * url, NSString * title, void (^)(NSObject<_WKWebExtensionBookmark> *, NSError *));
```

Comment:
Nit: Some extra spaces around the ` * `. It should be `NSString *parentId`, and the like for other ones.